tokenConfig.json

```diff
@@ -1,17 +1,11 @@
 {
+   "collectionID": "devcon",
    "onChain": false,
    "title": "Devcon",
-   "tokenName": "devcon",
+   "image": "https://raw.githubusercontent.com/TokenScript/token-negotiator/main/mock-images/devcon.svg",
```

Comment:
Let us for the version 2.0 release move this image outside of our core library and into the devcon outlet example.